docs/resources-overview.md

```diff
@@ -42,27 +35,31 @@ To see only objects of a specific type, for example to see the webhook and
 controller deployments inside Knative Serving, you can run
 `kubectl -n knative-serving get deployments`.
 
-The Knative Serving controller creates Kubernetes, Istio, and Build CRD
-resources when Knative Serving resources are created and updated. These
+The Knative Serving controller creates Kubernetes and Istio
+resources when Knative Serving resources are created and updated. It will also
+create Build resources when provided in the Configuration spec. These
 sub-resources will be created in the same namespace as their parent Knative
-Serving resource, _not_ the `knative-serving` namespace.
+Serving resource, _not_ the `knative-serving` namespace. For example, if you
+create a Knative Serivce in namespace 'foo' the cooresponding Istio resources
+will also be in namespace 'foo'.
 
 ## Kubernetes Resource Configs
 
 The various Kubernetes resource configurations are organized as follows:
 
 ```plain
 # Knative Serving resources
-config/*.yaml
+`config/*.yaml`
```

Comment:
This is in a preformatted block, so [this adds literal backticks around these items](https://github.com/knative/serving/blob/03ba10de1eb221bc35496ee203cc9d0c3427e4b8/docs/resources-overview.md#kubernetes-resource-configs).

Reply:
I see... Removed